next-tavla/app/(admin)/organizations/components/MemberAdministration/MemberList.tsx

```diff
@@ -12,16 +16,27 @@ function MemberList({
     oid?: TOrganizationID
 }) {
     return (
-        <div className="flexColumn g-1">
-            {members.map((member) => (
-                <div className={classes.memberListRow} key={member.uid}>
-                    <div>{member.email}</div>
-                    {member.uid !== currentUserId && (
-                        <RemoveUserButton uid={member.uid} oid={oid} />
-                    )}
-                </div>
-            ))}
-        </div>
+        <table className="flexColumn g-1 mt-3">
+            <thead>
+                <tr className={classes.tableHeader}>
+                    {MEMBER_TABLE_COLUMNS.map((column) => (
+                        <th key={column}>{MemberTableColumn[column]}</th>
+                    ))}
+                </tr>
+            </thead>
+            <tbody>
+                {members.map((member) => (
+                    <tr className={classes.memberListRow} key={member.uid}>
+                        <td>{member.email}</td>
+                        {member.uid !== currentUserId && (
+                            <td>
+                                <RemoveUserButton user={member} oid={oid} />
+                            </td>
+                        )}
+                    </tr>
+                ))}
+            </tbody>
+        </table>
```

Comment:
Jeg syntes ikke vi skal bruke <table> for denne typen data. Kompleksiteten blir økt uten nevneverdig økning i verdi